src/main/scala/org/opennetworkinsight/proxy/ProxyFeedback.scala

```diff
@@ -0,0 +1,86 @@
+package org.opennetworkinsight.proxy
+
+import org.apache.spark.SparkContext
+import org.apache.spark.rdd.RDD
+import org.apache.spark.sql.{DataFrame, SQLContext}
+import org.apache.spark.sql.types.{StructType, StructField, StringType}
+import org.apache.spark.sql.Row
+import scala.io.Source
+
+import org.opennetworkinsight.proxy.ProxySchema._
+
+object ProxyFeedback {
+
+  case class Feedback(date: String,
+                      timeStamp: String,
+                      clientIP: String,
+                      host: String,
+                      reqMethod: String,
+                      userAgent: String,
+                      responseContentType: String,
+                      respCode: String,
+                      fullURI: String) extends Serializable
+
+
+  def loadFeedbackDF(feedbackFile: String,
+                     duplicationFactor: Int,
+                     sc: SparkContext,
+                     sqlContext: SQLContext): DataFrame = {
+
+    import org.apache.spark.sql._
```

Comment:
Do we need more stuff from org.apache.spark.sql._ other than DataFrame, SQLContext, StructType, StructField and StringType? If we do I'd move this import to the top level, else I'd remove it.